lib/utils/normalizeOptions.js

```diff
@@ -9,6 +9,12 @@ function normalizeOptions(compiler, options) {
   options.contentBase =
     options.contentBase !== undefined ? options.contentBase : process.cwd();
 
+  // Setup default value
+  options.contentBasePublicPath =
```

Comment:
Please add a case to test when this option is set in `normalizeOptions.test.js`.